vassal-app/src/main/java/VASSAL/counters/GamePiece.java

```diff
@@ -57,19 +90,20 @@
    */
   public Point getPosition();
 
+  /** @param p Sets the location of this piece on its {@link Map} */
   public void setPosition(Point p);
 
   /**
-   * The area which this GamePiece occupies when
-   * drawn at the point (0,0)
+   * @return The area which this GamePiece occupies when drawn at the point (0,0)
    */
   public Rectangle boundingBox();
 
   /**
-   * The shape of the piece from the user's viewpoint.  This defines the area
+   * @return The shape of the piece from the user's viewpoint. This defines the area
    * in which the user must click to select or move the piece, for example.
    * Like {@link #boundingBox}, it assumes the position is (0,0) and must be translated
-   * to the actual location where the piece is being drawn.
+   * to the actual location where the piece is being drawn. For most ordinary pieces, the
```

Comment:
True, but (0, 0) correspoinds to the centre of a piece, not the top-left corner as you might expect. Worth mentioning.